docs/pages/programming/c_api.md

```diff
@@ -325,6 +326,37 @@ Parameters:
 |*valueaddr*| input | The value pointer of replaceable param | The value associated with this pointer should not change between bind and [cdb2_run_statement](#cdb2runstatement), and for numeric types must be signed. |
 |*length*| input | The length of replaceable param | This should be the sizeof(valueaddr's original type), so 1 if it's a char, 4 for float... |
 
+### cdb2_bind_index
+```
+int cdb2_bind_index(cdb2_hndl_tp *hndl, int index, int type, const void *varaddr, int length);
+```
+
+Description:
+
+This routine is used to bind value pointers to replaceable params in sql statement. The index starts from 1, and increases for every new parameter in the statement. This version of cdb2_bind_* is faster than cdb2_bind_param.
```

Comment:
This routine is used to bind value pointers to replaceable params in sql statement. ->
This routine is used to bind value pointers to replaceable params based on their position/index in the SQL statement.